app/Models/Permission.php

```diff
@@ -203,6 +205,13 @@ class Permission extends Model
             ],
         ],
 
+        'logs' => [
+            'description' => 'Permissions that control a user\'s access to the logs for this server.',
+            'keys' => [
+                'see' => 'Allows a user to see logs.',
```

Comment:
should be `read` to follow consistency and also what you used in the ACTION_LOGS_READ const